deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -442,17 +442,25 @@ become_leader0(QName, Name) ->
 all_replica_states() ->
     Rows0 = ets:tab2list(ra_state),
     Rows = lists:map(fun
-                         ({K, follower, promotable}) ->
-                             {K, promotable};
-                         ({K, follower, non_voter}) ->
-                             {K, non_voter};
-                         ({K, S, _}) ->
-                             %% voter or unknown
-                             {K, S};
-                         (T) ->
-                             T
+                        (T = {K, _, _}) ->
+                            case rabbit_process:is_registered_process_alive(K) of
```

Comment:
`whereis/1` will do here, you are already on the node where the process is running.

Reply:
Ok, i avoided the `rabbit_process:is_process_alive/1` which does the extra distribution work. But also noticed we have `rabbit_process:is_registered_process_alive/1` which is already doing `whereis/1` with `is_pid/1` guard check on the [local node](https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbit/src/rabbit_process.erl#L77-L78), which i'd still have to do on the case statement. seems to be doing same thing (if i'm seeing correct)

Reply:
```case whereis(K) of
         undefined -> ...
         Pid -> to_replica_state(T)
    end
```

I'm not sure using a helper function for the above is worth it, just causes read indirection.

Reply:
k cool, definitely easier to read. sorry for the back-&-forth on this small item (tendency has been check/use rabbit helpers 1st 😊 after finding myself rewriting some existing utils in the past 😅 ). Yep, the read indirection not necessary here 👍